scripts/autocomplete.sh

```diff
@@ -76,7 +76,7 @@ def process_function(namespace, element):
 
 def process_dir_globals(target):
     infile = os.path.join(base_dir, target, src_filename)
-    with open(infile) as f:
+    with open(infile, encoding="utf-8") as f:
```

Comment:
On Windows, the `open` encoding depends on locale, which means opening the file is a lottery.

This should become default in Python 3.15 [1]. However, we have to force it for now.

[1] See: https://peps.python.org/pep-0686/